test/expected/rowsecurity-11.out

```diff
@@ -3475,34 +3445,31 @@ SET ROLE regress_rls_bob;
 PREPARE role_inval AS SELECT * FROM t1;
 -- Check plan
 EXPLAIN (COSTS OFF) EXECUTE role_inval;
-          QUERY PLAN           
--------------------------------
- Append
-   ->  Seq Scan on t1
-         Filter: ((a % 2) = 0)
-(3 rows)
+        QUERY PLAN        
```

Comment:
we probably do want to add some data to this table so that we can see the filter correctly applied. Right now these test no longer test anything.